config.js

```diff
@@ -1,30 +1,32 @@
-
 /**
  * Module dependencies.
  */
-var express = require('express'),
-    favicon = require('serve-favicon'),
-    logger = require('morgan'),
-    bodyParser = require('body-parser'),
-    methodOverride = require('method-override'),
-    errorHandler = require('errorhandler'),
-    http = require('http'),
-    path = require('path');
+import express from 'express';
```

Comment:
Probably you need to add a js linter to the project as eslint 
https://nextjs.org/docs/basic-features/eslint

Reply:
Normally semicolons are not recommended in imports.

Reply:
Added eslint